portal/models/tou.py

```diff
@@ -7,9 +7,10 @@
 from .organization import Organization, OrgTree
 from .role import ROLE, Role
 from .user import User
-
+#TODO check if it is right to add the extra TOU type here? Migration needed then?
```

Comment:
Yes, this requires a migration, to update the enumeration in the db.  I can help w/ that later if you would like.

Reply:
thank you, @pbugni Your help in writing the migration script is much appreciated.